ntoskrnl/ex/time.c

```diff
@@ -30,6 +30,115 @@ ULONG ExpTimerResolutionCount = 0;
 
 /* FUNCTIONS ****************************************************************/
 
+/*++
+ * If successful, this function sets the following Global Variables:
+ * ExpTimeZoneInfo
+ * ExpTimeZoneBias
+ * ExpTimeZoneId
+ *
+ * It returns *TimeZoneId as follows:
+ *     0 = TIME_ZONE_ID_UNKNOWN
+ *     1 = TIME_ZONE_ID_STANDARD
+ *     2 = TIME_ZONE_ID_DAYLIGHT
```

Comment:
Why should the actual numerical values matter?

Reply:
If someone wanted to print out a single value, then could decode it based on this table.

Reply:
They should look at the defines directly:
https://github.com/search?q=repo%3Areactos%2Freactos%20TIME_ZONE_ID_STANDARD&type=code
Duplicating the values (again) is not good.

Reply:
@SergeGautherie, I have spent many hours lately modifying and trouble-shooting this code.
In doing so, I have liberally added statements such as the following to my testing code:
```
    DPRINT1("*TimeZoneId is %d\n", *TimeZoneId);
    DPRINT1("ExpTimeZoneId is %d\n", ExpTimeZoneId);
```
And, I know that technically the print specifier above is incorrect.
So knowing and using these values by their absolute numeric values was very helpful to me.
Therefore, I would hope to help any others that come to this code in the future.
But in the interest of keeping peace here, I will delete this section of code.

Reply:
Code section deleted. Thanks.